x-pack/plugins/siem/public/pages/overview/translations.ts

```diff
@@ -14,17 +14,14 @@ export const PAGE_SUBTITLE = i18n.translate('xpack.siem.overview.pageSubtitle',
   defaultMessage: 'Security Information & Event Management with the Elastic Stack',
 });
 
-export const NO_FILEBEAT_INDICES = i18n.translate(
-  'xpack.siem.overview.network.noFilebeatIndicies',
-  {
-    defaultMessage: "Looks like you don't have any Filebeat and Auditbeat indices.",
-  }
-);
+export const EMPTY_TITLE = i18n.translate('xpack.siem.overview.emptyTitle', {
+  defaultMessage: "It looks like you don't have any indices relevant to the SIEM application",
```

Comment:
nit: single quotes rather than double quotes

Reply:
It was double quotes because of the "apostrophe" in `don't`. But technically it's not an apostrophe at all; it's an inches symbol. Typography nerd stuff aside, will switch to a proper typographic apostrophe and back to single quotes :)